### PR TITLE
update load base split config

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by qiancai** [comment]
@HunDunDM and @nolouch, PTAL and share your comments. Thanks!

**Review by nolouch — APPROVED** [review]
lgtm

**Review by HunDunDM — APPROVED** [review]

**Comment by TomShawn** [comment]
@yikeke Please review this PR, thanks!

File: configure-load-base-split.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
 
 ## 使用方法
 
-目前的 Load Base Split 的控制参数为 `split.qps-threshold`。这个阈值的含义是：如果连续 10s 内，某个 Region 每秒的各类读请求之和超过 `split.qps-threshold`，那么就对此 Region 进行拆分。
+目前的 Load Base Split 的控制参数为 `split.qps-threshold` 和 `split.byte-threshold`。这两个阈值的含义是：如果连续 10s 内，某个 Region 每秒的各类读请求之和都超过其中任意一个阈值，那么就对此 Region 进行拆分。
 
-目前默认开启 Load Base Split，但配置相对保守，默认为 `3000`。如果想要关闭这个功能，可以将这个阈值调到足够高即可。
+目前默认开启 Load Base Split，但配置相对保守，`split.qps-threshold` 默认为 `3000`，`split.byte-threshold` 默认为 30MB/s。如果想要关闭这个功能，可以将这两个阈值全部都调到足够高即可。
```
Comment (yikeke) [inline configure-load-base-split.md:34]:
```suggestion
目前默认开启 Load Base Split，但配置相对保守，`split.qps-threshold` 默认为 `3000`，`split.byte-threshold` 默认为 30MB/s。如果想要关闭这个功能，将这两个阈值调到足够高即可。
```

File: dynamic-config.md [hunk 2]
```diff
@@ -193,9 +193,10 @@ show warnings;
 | {db-name}.{cf-name}.titan.blob-run-mode | 处理 blob 文件的模式 |
 | storage.block-cache.capacity | 共享 block cache 的大小（自 v4.0.3 起支持） |
 | backup.num-threads | backup 线程的数量（自 v4.0.3 起支持） |
-| split.qps-threshold | 对 Region 执行 load-base-split 的阈值。如果读 QPS 连续 10 秒内均超过这个值，则进行 split |
-| split.split-balance-score | load-base-split 的控制参数，确保 split 后左右访问尽量均匀 |
-| split.split-contained-score | load-base-split 的控制参数，尽量减少 split 后跨 Region 访问 |
+| split.qps-threshold | 对 Region 执行 load-base-split 的阈值。如果连续一段时间内，某个 Region 的读请求的 QPS 超过 qps-threshold，则进行 split |
+| split.byte-threshold | 对 Region 执行 load-base-split 的阈值。如果连续一段时间内，某个 Region 的读请求的流量超过 byte-threshold，则进行 split |
```
Comment (yikeke) [inline dynamic-config.md:197]:
```suggestion
| split.qps-threshold | 对 Region 执行 load-base-split 的阈值。如果连续一段时间内，某个 Region 的读请求的 QPS 超过 qps-threshold，则切分该 Regiont |
| split.byte-threshold | 对 Region 执行 load-base-split 的阈值。如果连续一段时间内，某个 Region 的读请求的流量超过 byte-threshold，则切分该 Region |
```

File: dynamic-config.md [hunk 3]
```diff
@@ -193,9 +193,10 @@ show warnings;
...
+| split.split-balance-score | load-base-split 的控制参数，确保 split 后左右访问尽量均匀，数值越小越均匀，但也可能导致无法切分 |
+| split.split-contained-score | load-base-split 的控制参数，数值越小 split 后跨 Region 的访问越少 |
```
Comment (yikeke) [inline dynamic-config.md:199]:
```suggestion
| split.split-balance-score | load-base-split 的控制参数，确保 Region 切分后左右访问尽量均匀。数值越小越均匀，但也可能导致无法切分 |
| split.split-contained-score | load-base-split 的控制参数，数值越小，Region 切分后跨 Region 的访问越少 |
```

File: configure-load-base-split.md [hunk 4]
```diff
@@ -29,9 +29,9 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
 
 ## 使用方法
 
-目前的 Load Base Split 的控制参数为 `split.qps-threshold`。这个阈值的含义是：如果连续 10s 内，某个 Region 每秒的各类读请求之和超过 `split.qps-threshold`，那么就对此 Region 进行拆分。
+目前的 Load Base Split 的控制参数为 `split.qps-threshold` 和 `split.byte-threshold`。这两个阈值的含义是：如果连续 10s 内，某个 Region 每秒的各类读请求之和都超过其中任意一个阈值，那么就对此 Region 进行拆分。
```
[the remark this answers is not in the document]
Comment (HunDunDM) [reply configure-load-base-split.md:32]: 各类读请求的数量之和超过阈值或流量之和超过阈值

File: configure-load-base-split.md [hunk 5]
```diff
@@ -29,9 +29,9 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
 
 ## 使用方法
 
-目前的 Load Base Split 的控制参数为 `split.qps-threshold`。这个阈值的含义是：如果连续 10s 内，某个 Region 每秒的各类读请求之和超过 `split.qps-threshold`，那么就对此 Region 进行拆分。
+目前的 Load Base Split 的控制参数为 `split.qps-threshold` 和 `split.byte-threshold`。这两个阈值的含义是：如果连续 10s 内，某个 Region 每秒的各类读请求之和都超过阈值(QPS阈值或流量阈值)，那么就对此 Region 进行拆分。
```
Comment (yikeke) [inline configure-load-base-split.md:32]:
```suggestion
目前的 Load Base Split 的控制参数为 `split.qps-threshold`（QPS 阈值）和 `split.byte-threshold`（流量阈值）。如果连续 10s 内，某个 Region 每秒的各类读请求之和都超过 QPS 阈值或流量阈值，那么就对此 Region 进行拆分。
```

File: dynamic-config.md [hunk 6]
```diff
@@ -193,9 +193,10 @@ show warnings;
 | {db-name}.{cf-name}.titan.blob-run-mode | 处理 blob 文件的模式 |
 | storage.block-cache.capacity | 共享 block cache 的大小（自 v4.0.3 起支持） |
 | backup.num-threads | backup 线程的数量（自 v4.0.3 起支持） |
-| split.qps-threshold | 对 Region 执行 load-base-split 的阈值。如果读 QPS 连续 10 秒内均超过这个值，则进行 split |
-| split.split-balance-score | load-base-split 的控制参数，确保 split 后左右访问尽量均匀 |
-| split.split-contained-score | load-base-split 的控制参数，尽量减少 split 后跨 Region 访问 |
+| split.qps-threshold | 对 Region 执行 load-base-split 的阈值。如果连续一段时间内，某个 Region 的读请求的 QPS 超过 qps-threshold，则切分该 Region |
+| split.byte-threshold | 对 Region 执行 load-base-split 的阈值。如果连续一段时间内，某个 Region 的读请求的流量超过 byte-threshold，则切分该 Region |
+| split.split-balance-score | load-base-split 的控制参数，确保 Region 切分后左右访问尽量均匀，数值越小越均匀，但也可能导致无法切分 |
+| split.split-contained-score | load-base-split 的控制参数，Region 切分后跨 Region 的访问越少 |
```
Comment (yikeke) [inline dynamic-config.md:199]:
```suggestion
| split.split-contained-score | load-base-split 的控制参数，数值越小，Region 切分后跨 Region 的访问越少 |
```

**Comment by qiancai** [comment]
@Joyinqin This doc changes in this PR is ready for EN PR creation, PTAL. Thanks~

**Pull request merged**